core/src/main/java/org/frankframework/pipes/AbstractPipe.java

```diff
@@ -257,19 +259,19 @@ public PipeForward findForward(@Nullable String forward){
 	}
 
 	@Override
-	public Map<String, PipeForward> getForwards(){
-		Map<String, PipeForward> forwards = new Hashtable<>(pipeForwards);
+	@Nonnull
+	public Map<String, PipeForward> getForwards() {
+		Map<String, PipeForward> forwards = new HashMap<>(pipeForwards);
 		PipeLine pipeline = getPipeLine();
-		if (pipeline==null) {
-			return null;
+		if (pipeline == null) {
+			return forwards;
 		}
 
 		//Omit global pipeline-forwards and only return local pipe-forwards
 		List<IPipe> pipes = pipeline.getPipes();
```

Comment:
I think you can do this even more compact in a stream:

```suggestion
    pipeline.getPipes()
				.forEach(pipe -> forwards.remove(pipe.getName()));
```

Reply:
Will do!